torch/distributed/distributed_c10d.py

```diff
@@ -1714,6 +1715,10 @@ def broadcast_object_list(object_list, src=0, group=None):
         src (int): Source rank from which to broadcast ``object_list``.
         group: (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used. Default is ``None``.
+        map_location (torch.device): The device to load tensors contained in
```

Comment:
how does this render? Do we need double quote to wrap `torch.device`?

Reply:
![broadcast_object_list_render](https://user-images.githubusercontent.com/31054793/125491626-59c10080-df5c-44df-a38f-ac6887578f3f.png)
In my minimal experience, I have found that as long as the type in the parentheses is a real class, then the render will be fine without the wrapping double quotes.